src/shared/cloudformation/cloudformation.ts

```diff
@@ -13,6 +13,7 @@ import { getLogger } from '../logger'
 export namespace CloudFormation {
     export const SERVERLESS_FUNCTION_TYPE = 'AWS::Serverless::Function'
     export const LAMBDA_FUNCTION_TYPE = 'AWS::Lambda::Function'
+    export const LAMBA_API_TYPE = 'AWS::Lambda::Api'
```

Comment:
Does `AWS::Serverless::Api` work?

Reply:
That was a typo, that is what this is supposed to be, fixed